src/chains/ethereum/ethereum/tests/provider.test.ts

```diff
@@ -30,6 +30,154 @@ describe("provider", () => {
         });
       });
     });
+
+    it("uses the default 'clock' time for the `timestampIncrement` option", async () => {
+      const provider = await getProvider();
+      const options = provider.getOptions();
+      assert(options.miner.timestampIncrement, "clock");
+
+      const accounts = await provider.send("eth_accounts");
+      const tx = { from: accounts[0], gas: "0xfffff" };
+      const sentAfter = Math.floor(Date.now() / 1000);
```

Comment:
The `sentAfter`/`sentBefore` variable names here are very confusing to me. Maybe consider naming them after what the time represents rather than how the block's timestamp should compare to it? Something like `timeBeforeMiningBlock`/`timeAfterMiningBlock`?

That will also allow the failure statements to read like a sentence:
> block wasn't mined at the right time, should have been on or after ${timeBeforeMiningBlock}